Jenkinsfile

```diff
@@ -1,3 +1,5 @@
+@Library('pipeline-library@pull/784/head') _
+
```

Comment:
```suggestion
```
To be removed in a future PR when https://github.com/jenkins-infra/pipeline-library/pull/784 will be merged.